MAUI/Carousel-View/Animation.md

```diff
@@ -0,0 +1,119 @@
+---
+layout : post
+title: Animation in .NET MAUI Carousel View control | Syncfusion
+description: Learn here all about Animation support in Syncfusion .NET MAUI Carousel View (SfCarousel) control and more.
+platform : maui
+control : Carousel
+documentation : ug
+---
+
+# Animation in .NET MAUI Carousel View (SfCarousel)
+
+The `Duration` property of the `SfCarousel` control specifies the time taken to move an item to the selected item position. The duration is specified in seconds. The default value is 600 ms.
```

Comment:
The duration is specified in seconds. or milli seconds?

Reply:
milli seconds sir.